apps/platform/src/app/auth/otp/page.tsx

```diff
@@ -8,6 +8,8 @@ import Cookies from 'js-cookie'
 import { toast } from 'sonner'
 import { LoadingSVG } from '@public/svg/shared'
 import { KeyshadeBigSVG } from '@public/svg/auth'
+import type { ClientResponse } from '@keyshade/api-client/dist/src/types/index.types'
+import type { ResendOTPRequest } from '@keyshade/api-client/dist/src/types/auth.types'
```

Comment:
I don't think we should allow this import style. The types should come from `@keyshade/api-client/types/<type file>`

Reply:
yup i wanted that manner only but this was just a workaround to show you how that  2nd solution would work which I mentioned from stackoverflow.

Reply:
the issue is @keyshade/api-client is a package and points to `dist/src/index.js` in the api-client when we import it as written in the `package.json`. And hence the syntax suggested by you would work only if the some different path is assigned there or by some strategy. Any Suggestions?